itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenarios.java

```diff
@@ -2518,11 +2518,16 @@ public void testRenamePartitionedTableAcrossDatabases() throws IOException {
 
   @Test
   public void testViewsReplication() throws IOException {
+    boolean verify_setup_tmp = VERIFY_SETUP_STEPS;
```

Comment:
you can add a new test just for MV. 

Create MV with same DB and diff DB
Test after bootstrap, replica shouldn't have MV.

Create MV with same DB and diff DB
Test after incremental, replica shouldn't have MV.